upgrade/lab6.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "ACTIVATING SCENARIO #6...
+"
+
+echo '' | sudo tee /etc/resolv.conf >/dev/null 2>&1
```

Comment:
I would just move the original resolv.conf elsewhere... otherwise, how are they going to fix it / rollback?